archlinux/customization/hooks/00-alarm-uid-gid.chroot

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+# Change uid/gid of alarm user
```

Comment:
minor comment, add note saying that this change is required to not conflict with android_system user.